content/blog/youtrack-github.md

```diff
@@ -0,0 +1,36 @@
+---
+author: "Alex Hill"
+date: 2019-04-17
```

Comment:
Should we pick a date less likely to be buried - perhaps for early after Easter?